test/test-unit.js

```diff
@@ -355,6 +354,17 @@ suite('GaiaHeader', function() {
         sinon.assert.calledOnce(this.fontFit);
       });
     });
+
+    test('children with .l10n-action are not considered when calculating title space', function() {
+      this.dom.innerHTML = '<gaia-header action="back"><span l10n-action>' +
+        '</span><h1>Title title</h1></gaia-header>';
+      var header = this.dom.firstElementChild;
+      var h1 = header.querySelector('h1');
+
+      return afterNext(header, 'runFontFit').then(() => {
+        assert.equal(h1.style.marginInlineStart, '-50px');
+      });
+    });
```

Comment:
mmm this test is not failing on master. Can you do a test failing in master, but passing with your patch ?